Lib/fontTools/varLib/__init__.py

```diff
@@ -130,16 +132,27 @@ def _add_fvar(font, axes, instances: List[InstanceDescriptor]):
         psname = instance.postScriptFontName
 
         inst = NamedInstance()
-        inst.subfamilyNameID = nameTable.addMultilingualName(
-            localisedStyleName, mac=macNames
+        inst.coordinates = {
+            axes[k].tag: axes[k].map_backward(v) for k, v in coordinates.items()
+        }
+
+        subfamilyNameID = nameTable.addMultilingualName(
```

Comment:
I find it a bit confusing that that you call a method named `addMultilingualName` twice. I know this only adds if it one is not already there, but wouldn't it read better if you did findMultilingualName and then add... (the latter only once)?

Reply:
Maybe I'm being dense, but how? `Find` can also return None, so you'd have to have two calls to `add` either way? Like so:

```py
subfamilyNameID = findMultilingualName(...)
if subfamilyNameID in {2, 17} and inst.coordinates != default_coordinates:
    inst.subfamilyNameID = addMultilingualName(..., minNameID=256)
elif subfamilyNameID is None:
    inst.subfamilyNameID = addMultilingualName(...,  minNameID=0)
else:
    inst.subfamilyNameID = subfamilyNameID
```

Reply:
would this not work?

```python
subfamilyNameID = findMultilingualName(...)
if subfamilyNameID in {2, 17} and inst.coordinates == default_coordinates:
    inst.subfamilyNameID = subfamilyNameID
else:
    # subfamilyNameID is None, or neither 2 or 17, or it is but instance coordinates != default
    inst.subfamilyNameID = nameTable.addMultilingualName(
        localisedStyleName, windows=True, mac=macNames, minNameID=256
    )
```